packages/gen-ai-hub/package.json

```diff
@@ -23,12 +23,14 @@
     "test": "NODE_OPTIONS=--experimental-vm-modules jest",
     "lint": "eslint . && prettier . --config ../../.prettierrc --ignore-path ../../.prettierignore -c",
     "lint:fix": "eslint . --fix && prettier . --config ../../.prettierrc --ignore-path ../../.prettierignore -w --log-level error",
-    "generate": "openapi-generator -i ./src/orchestration/spec/ -o ./src/orchestration/client"
+    "generate:orchestration": "openapi-generator --options-per-service='./src/orchestration/spec/options-per-service.json' --overwrite --generateESM -i ./src/orchestration/spec/api.yaml -o ./src/orchestration && pnpm lint:fix",
+    "postgenerate": "ts-node ../../scripts/update-imports.ts src/orechestration/api"
```

Comment:
[q] We dont need this command here and the script now right?

Reply:
Good catch, that was just a leftover from my previous approach 😅